src/System.ComponentModel.Composition/tests/System/ComponentModel/Composition/Hosting/AssemblyCatalogTests.cs

```diff
@@ -109,6 +109,14 @@ public static void Constructor_InvalidFileNameAsCodeBaseArgument_ShouldThrowArgu
             });
         }
 
+        public static void Constructor_InvalidFileNameAsCodeBaseArgument_ShouldThrowIO(Func<string, AssemblyCatalog> catalogCreator)
+        {
+            Assert.ThrowsAny<IOException>(() =>
+            {
+                var catalog = catalogCreator("??||>");
+            });
```

Comment:
Nit: doesn't need to store the result:
```C#
Assert.ThrowsAny<IOException>(() => catalogCreator("??||>"));
```

Reply:
I was following along with the rest of the file's style. I'll go ahead and change all of them.